cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,66 @@
+// for local conduit only
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
 
+import settingsPageObject from '../support/pages/settings.page-object'
+const settingsPage = new settingsPageObject()
+describe('Settings page', () => {
+  let user
   beforeEach(() => {
-
-  });
-
+    cy.task('db:clear')
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser
+      cy.login(user.email, user.username, user.password)
+    })
+  })
   it('should provide an ability to update username', () => {
+    cy.visit('/settings')
+    settingsPage.usernameField.type('new')
+    settingsPage.updateBtn.click()
 
-  });
+    // settingsPage.modalText.should('contain', 'Update successful!')
+    settingsPage.usernameField.should('have.value', user.username + 'new')
+  })
 
   it('should provide an ability to update bio', () => {
+    settingsPage.visit()
+    settingsPage.bioField.type('new')
```

Comment:
```suggestion
    settingsPage.bioField.type('new')
```
wrap this and similar method into page object methods like  settingsPage.fillThebioField('new');